### PR TITLE
package for tf serving

### PR DESCRIPTION
**Pull request by Orion34-lanbo**
Support sdk packing for tensorflow side, which can be used for tensorflow-serving build with `DISC`. 
Tested in internal `unibuild` pipeline.

### CODE REVIEW

File: scripts/python/tao_build.py [hunk 1]
```diff
@@ -688,7 +688,22 @@ def add_to_tar(tar, file, dir_in_tar="", name_in_tar=""):
...
                 add_to_tar(tar, libstdcxx_path, name_in_tar=libstdcxx_name)
-        logger.info("sdk package created   : " + dsw_tgz)
+        logger.info("sdk package created : " + dsw_tgz)
+
+        # pkg for tf serving
+        serving_tgz = "{}/tao/tao_serving_sdk_{}.tgz".format(root, args.version)
+        python_bin = os.path.join(args.venv_dir, "bin", "python3")
+        tf_major, _, _, _, _, _, _, _ = get_tf_info(python_bin)
+        # patchelf to remove libtensorflow_framework.so.* deps
+        patchelf_cmd = f"patchelf --remove-needed libtensorflow_framework.so.{tf_major} {F_TAO_OPS_SO}"
```
Comment (qiuxiafei) [inline scripts/python/tao_build.py:698]: May have unwanted side-effect that the shared library will be left unusable for cases other than tfserving... consider making a copy?

Comment (Orion34-lanbo) [reply scripts/python/tao_build.py]: change to use `genrule` to generate so file in different dir.

File: tao/BUILD [hunk 2]
```diff
@@ -140,6 +140,18 @@ tf_cc_shared_object(
...
 
+genrule(
+    name = "lib_tao_serving_genrule",
+    srcs = [
+        ":libtao_ops.so"
+    ],
+    outs = ["serving/libtao_ops.so"],
+    cmd = """
+        cp $(location :libtao_ops.so) $@;
+        patchelf --remove-needed libtensorflow_framework.so.%s $@
```
Comment (qiuxiafei) [inline tao/BUILD:151]: nice change!

**Review by qiuxiafei — APPROVED** [review]

**Review by qiuxiafei — APPROVED** [review]

**Review by qiuxiafei — APPROVED** [review]
LGTM

**Pull request merged**